starcheck/utils.py

```diff
@@ -245,9 +248,18 @@ def _yagzag_to_pixels(yag, zag):
     return row.tolist(), col.tolist()
 
 
-def _guide_count(mags, t_ccd, count_9th=False):
+def _t_ccds_bonus(mags, t_ccd, date):
```

Comment:
I'm not sure why `_t_ccds_bonus` and `_guide_count` are being named as private methods. They are effectively used as public elsewhere in the code.

Reply:
OK.  I renamed both methods.  I wasn't sure what to call either.  Hopefully guide_count() and apply_t_ccds_bonus() make some sense.